templates/hooks/open_id_connect_provider/sdk_read_one_post_set_output.go.tpl

```diff
@@ -0,0 +1,5 @@
+	if tags, err := rm.getTags(ctx, &resource{ko}); err != nil {
+		return nil, err
+	} else {
+		ko.Spec.Tags = tags
+	}
```

Comment:
We should mention this hook in the `generator.yaml` file in order to have injected in the generated code

Reply:
Done